tachyon/math/finite_fields/mersenne31/packed_mersenne31_avx2.cc

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) 2022 The Plonky3 Authors
+// Use of this source code is governed by a MIT/Apache-2.0 style license that
+// can be found in the LICENSE-MIT.plonky3 and the LICENCE-APACHE.plonky3
+// file.
+
+#include "tachyon/math/finite_fields/mersenne31/packed_mersenne31_avx2.h"
+
+#include <immintrin.h>
+
+#include "tachyon/math/finite_fields/packed_prime_field32_avx2.h"
+
+namespace tachyon::math {
+
+namespace {
+
+__m256i kP;
```

Comment:
(Optional) what about `kModulus` for clarity?

Reply:
the reason why I chose `P` are 2.

1. To match the original code as much as possible
2. The variable in the comment is a just single letter.